.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Why is this file included?

Reply:
including it documents (via the IDE) the linter settings and setup to run the unit tests in the IDE. As a general rule, I wouldn't include IDE specific settings, but given the broad usage of VS Code and the utility of including the file, I wanted to see what you all thought